aspnetcore/test/integration-tests.md

```diff
@@ -184,6 +184,71 @@ Because another test in the `IndexPageTests` class performs an operation that de
 
 [!code-csharp[](integration-tests/samples/2.x/IntegrationTestsSample/tests/RazorPagesProject.Tests/IntegrationTests/IndexPageTests.cs?name=snippet3)]
 
+### Set the environment
+
+If the SUT requires special testing configuration when tests are executed:
+
+* Provide test environment-specific configuration using one or both of the following approaches:
+  * Provide conditional configuration in the SUT's `Startup.ConfigureServices`/`Startup.Configure`:
+  
+    When configuring an `IWebHostBuilder` in *Startup.cs*:
+    
+    ```csharp
```

Comment:
Where is this code snippet, in configure services?

Reply:
Yes ... and given `IHostingEnvironment` examples we have are mostly shot into `Configure`, I'm not sure this is the best way to demo this. Should I go in the direction of a `Configure`-based injection and change this to ...

```csharp
public void Configure(IApplicationBuilder app, IHostingEnvironment env) 
{
    if (env.IsStaging())
    {
        // Code runs in the Staging environment
    }
}
```